wolfcrypt/ciphers.py

```diff
@@ -82,6 +82,39 @@
 ECC_BRAINPOOLP384R1 = 26
 ECC_BRAINPOOLP512R1 = 27
 
+RSA_PKCSV15_PAD = 0
```

Comment:
If I recall correctly, macro constants can be pulled from the underlying library directly. This definitely works for enums (see `static const long PRIVATEKEY_TYPE;` in _build_ffi.py for an example). If that works, we should do that, IMO, instead of repeating ourselves with the same constant definitions here.

Reply:
Ah! That wasn't done elsewhere so figured I'd follow the pattern. I'll fix it.